README.md

```diff
@@ -1,28 +1,33 @@
 # DevOps Apprenticeship: Project Exercise
 
-## Getting started
+## Running with Poetry
+This application has been migrated to work with poetry. In order to leverage this you will have to do a small amount of
+configuration and setup
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+### Install Poetry
 
-### On macOS and Linux
-```bash
-$ source setup.sh
-```
-### On Windows (Using PowerShell)
-```powershell
-$ .\setup.ps1
-```
-### On Windows (Using Git Bash)
-```bash
-$ source setup.sh --windows
-```
+#### Windows (Powershell)
+    (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
 
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
-```bash
-$ flask run
-```
+#### *nix (e.g. using BASH)
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+    
+Once this is done, run
+    
+    poetry --version
+to check that poetry was installed correctly. If not, you may have to make sure that poetry is added to your path.
 
-You should see output similar to the following:
+### Installing Dependencies with Poetry
+Please run
+
+    poetry add requests
+    poetry install
```

Comment:
No, the whole point of pyproject.toml is that it keeps track of all dependencies so you don't need to manually install any individual dependencies yourself. All you need to do is `poetry install`.

When you want to add a new package to your project, e.g. requests, you run `poetry add requests` one time, which should update pyproject.toml, and then you should commit that change. So then if someone else (or Vagrant) clones the repo and runs `poetry install`, it will install everything listed in pyproject.toml, including "requests".

You are also missing pytest and selenium. You should run `poetry install pytest selenium --dev`